torchquantum/functional.py

```diff
@@ -1168,6 +1169,9 @@ def singleexcitation_matrix(params):
     ),
     "ecr": torch.tensor(
         [[0, 0, 1, 1j], [0, 0, 1j, 1], [1, -1j, 0, 0], [-1j, 1, 0, 0]], dtype=C_DTYPE
+    ),
+    "sdg": torch.tensor(
+        [[1 + 0j, 0 + 0j],[0 + 0j, -0 - 1j]], dtype=C_DTYPE
```

Comment:
Could you change this to [[1, 0], [0, -1j]] for better readability?

Reply:
Sure!